tests/core/test_geodb.py

```diff
@@ -607,7 +605,7 @@ def make_test_df(self):
         df = pd.read_csv(file)
         df['geometry'] = df['geometry'].apply(wkt.loads)
         return df
-        # return GeoDataFrame(df, crs={'init': 'epsg:4326'}, geometry=df['geometry'])
+        # return Geopd.DataFrame(df, crs={'init': 'epsg:4326'}, geometry=df['geometry'])
```

Comment:
this looks strange. GeoDataFrame is among the imported ones. I think this was done by accidentally replacing all DataFrame with pd.DataFrame which also changed this occurance

Reply:
Oh dear. True. Well spotted!